nrf_security/Kconfig.tls

```diff
@@ -150,9 +150,24 @@ endif # MBEDTLS_SSL_PROTO_TLS1_2
 config MBEDTLS_DEBUG_C
 	bool
 	prompt "Enable the debug functions for TLS."
+	select CBPRINTF_PACKAGE_LONGDOUBLE
 	help
 	  Enable the debug functions for TLS.
 
+config MBEDTLS_DEBUG_LEVEL
```

Comment:
Where is this configuration used?

Reply:
Oh right. It already exists.

As far as I can tell you are only adding an additional dependency on the config.
So I think it would be more correct to do this:
```
config MBEDTLS_DEBUG_LEVEL
	int
	depends on MBEDTLS_DEBUG_C
```
Unless there is something I am missing about this?

Reply:
@plskeggs Any thoughts on this?

Also I saw an upstream PR that might be related to these changes: 
https://github.com/zephyrproject-rtos/zephyr/pull/46572/files

Reply:
Yes, I'm aware of that PR.  I had my own simpler PR https://github.com/zephyrproject-rtos/zephyr/pull/45003 which @rlubos suggested I should close.

Sorry I didn't see your comment "So I think it would be more correct to do this:" above.

Are you suggesting I should change this section of Kconfig.tls to **only** be the three lines:
```
config MBEDTLS_DEBUG_LEVEL
        int
        depends on MBEDTLS_DEBUG_C
```
and not the lines 160-169?

Reply:
> Are you suggesting I should change this section of Kconfig.tls to only be the three lines:

Yes.

Reply:
That doesn't work, because Zephyr's definition of MBEDTLS_DEBUG_LEVEL, which is in `zephyr/modules/mbedtls/Kconfig`, depends on `MBEDTLS_DEBUG`, which depends on `MBEDTLS_BUILTIN`.  In my case, I'm not using `MBEDTLS_BUILTIN`, I'm using `sdk-mbedtls`.

I would like to leave those lines 160-169.  Otherwise I would need to open another Zephyr PR to make the MBEDTLS_DEBUG_LEVEL apply to external mbedtls, and given that I opened this current PR many months ago, I'd rather not.

Reply:
Alright. No problem.